FAQ.md

```diff
@@ -44,9 +62,29 @@ If you don't have any followers, update your configuration and run
 Not really. For instance, the demonstration server [https://peertube.cpy.re](https://peertube.cpy.re) has 2 vCore and 2GB of RAM and consumes on average:
  * **CPU** -> nginx ~ 20%, peertube ~ 10%,   postgres ~ 1%, redis ~ 3%
  * **RAM** -> nginx ~ 6MB, peertube ~ 120MB, postgres ~ 10MB, redis ~ 5MB
- 
+
 So you would need:
  * **CPU** 1 core if you don't enable transcoding, 2 at least if you enable it
  * **RAM** 1GB
  * **Storage** Completely depends on how many videos your users will upload
 
+
+## Are you gonna use the Steem blockchain?
+
+Short answer: no, since like most appchains/votechains, it modifies the dynamic of creation, and as such cannot be integrated into mainline PeerTube. Read more about that in [the dedicated section](#what-is-creation-dynamic-and-why-not-modify-it).
+
+Long answer is that the Steem blockchain goes astray of its promises of fairness and decentralization: the deliberate relaunching of the currency to ensure centralization, and the stake-based voting power, makes manipulation by wealthy users inevitable ([source here](https://decentralize.today/the-ugly-truth-behind-steemit-1a525f5e156)). Worse, money generated goes primarily to stake holders ([source here](https://steemit.com/steemit/@orly/how-the-steem-pyramid-scheme-really-works) ). For more information, read the complete whitepaper analysis done by [Tone Vays](https://twitter.com/ToneVays/status/761975587451928576).
```

Comment:
nit: `primarily goes` (invert adverb and verb)

Reply:
nit: `stakeholders` in one word

Reply:
ah indeed! thanks :)